README.md

```diff
@@ -27,53 +22,196 @@ ember install ember-a11y-testing
+  rules: {
+    region: { enabled: true },
+  },
+  checks: {
+    'color-contrast': {
+      options: {
+        noScroll: true,
+      },
+    },
+  },
+});
+```
+
+Test module level:
 
-Ember A11y Testing also provides a simple helper to run accessibility audits
-on-demand within your test suite.
+```javascript
+import { module, test } from 'qunit';
+import { setRunOptions } from 'ember-a11y-testing';
 
-_Note:_ any tests run with Ember A11y Testing will adjust the testing container
+module('some test module', function (hooks) {
+  hooks.beforeEach(function () {
+    setRunOptions({
+      rules: {
+        region: { enabled: true },
+      },
+      checks: {
+        'color-contrast': {
+          options: {
+            noScroll: true,
+          },
+        },
+      },
+    });
+  });
+
+  // ...
+});
+```
+
+Individual test level:
+
+```javascript
+import { module, test } from 'qunit';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+import { setRunOptions } from 'ember-a11y-testing';
+
+module('some test module', function (hooks) {
+  test('some test', function (assert) {
+    setRunOptions({
+      rules: {
+        region: { enabled: true },
+      },
+      checks: {
+        'color-contrast': {
+          options: {
+            noScroll: true,
+          },
+        },
+      },
+    });
+
+    // ...
+    a11yAudit();
+    // ...
+  });
+
+  // ...
+});
+```
+
+When using `setRunOptions` during a test, the options you set are automatically reset when the test completes.
+
+### `a11yAudit` Usage
+
+`ember-a11y-testing` also provides a test helper to run accessibility audits
+on specific tests within your test suite.
+
+_Note:_ any tests run with `ember-a11y-testing` will adjust the testing container
 to occupy the entire screen. This is to simulate the actual application
 environment, as browsers adjust styles at small sizes for accessibility reasons.
 It will reset itself at the conclusion of testing though.
 
-#### Acceptance Tests
+#### Application Tests
 
-For Acceptance tests, the helper is an async test helper so you can use it like
-this:
+For Application tests, the helper is an async test helper which can be used in a similar fashion to other `@ember/test-helpers` helpers:
 
 ```javascript
-import a11yAudit from 'ember-a11y-testing/test-support/audit';
+import { visit } from '@ember/test-helpers';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
 
-// ...elided for brevity
+module('Some module', function () {
+  //...
 
-test('Some test case', async function (assert) {
-  await visit('/');
-  await a11yAudit();
-  assert.ok(true, 'no a11y errors found!');
+  test('Some test case', async function (assert) {
+    await visit('/');
+    await a11yAudit();
+    assert.ok(true, 'no a11y errors found!');
+  });
 });
 ```
 
-If your app does not allow async/await, you will need to use the the andThen() helper and a regular function (as opposed to `async function`).
+#### Integration and Unit Tests
```

Comment:
Can we unify these sections (the ones for `Application Tests` and `Integration / Unit Tests`)? There really isn't anything unique to any of the types...

Reply:
Yep.